src/locales/en.json

```diff
@@ -1,7 +1,8 @@
 {
   "APP": {
     "NAME": "Outdoor Exercise Map",
-    "ABOUT": "Information on the service"
+    "ABOUT": "Information on the service",
+    "DESCRIPTION": ""
```

Comment:
I guess these are coming later?

Reply:
Yup. I'll add values that I've translated myself if the official translations haven't arrived by the end of today.